example_problems/fltcmp/submissions/accepted/fltcmp-test-write-files-between-testcases.R

```diff
@@ -0,0 +1,70 @@
+# This checks if we can write extra files
+# between different testcase runs. R uses
+# a temporary directory which DOMjudge should
+# remove between runs and in case this is forgotten
```

Comment:
I see where the dir is created, but where is it removed?

Reply:
So I rechecked my filesystem and the files are indeed not removed. The test works because we get a different workingdir so the files can't be accessed anymore.

I wonder if we should also do the cleanup as extra security measure.

Reply:
Opened https://github.com/DOMjudge/domjudge/pull/2550 for that,